mergify_engine/branch_updater.py

```diff
@@ -102,7 +103,8 @@ async def pre_rebase_check(ctxt: context.Context) -> None:
 async def _do_rebase(
     ctxt: context.Context,
     user: user_tokens.UserTokensUser,
-    bot_account_feature: subscription.Features,
+    committer_name: str,
+    committer_email: str,
```

Comment:
```suggestion
    user: GitHubUser,
```
I think that would prevent repeating the `@users.noreply.github.com` in different places and the `name or login` in different places too.